cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js

```diff
@@ -0,0 +1,61 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
+import permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import devTeams from '../../../support/dictionary/devTeams';
+import Users from '../../../support/fragments/users/users';
+
+let user;
+
+describe('bulk-edit', () => {
+  before('create user', () => {
+    cy.createTempUser([
+      permissions.bulkEditView.gui,
+      permissions.bulkEditEdit.gui,
+      permissions.bulkEditCsvEdit.gui,
+      permissions.bulkEditCsvView.gui,
+      permissions.uiInventoryViewCreateEditItems.gui,
+      permissions.uiUsersView.gui,
+    ]).then((userProperties) => {
+      user = userProperties;
+      cy.login(user.username, user.password, {
+        path: TopMenu.bulkEditPath,
+        waiter: BulkEditSearchPane.waitLoading,
+      });
+    });
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C350670 Verify radio buttons on the Record types accordion (TaaS)',
```

Comment:
please add team name as well, like so: 'C350670 Verify radio buttons on the Record types accordion (firebird) (TaaS)'

Reply:
C350670 Verify radio buttons on the Record types accordion (Vega) (TaaS)

Reply:
wrong team, please see test case. 
![image](https://github.com/folio-org/stripes-testing/assets/72025248/e7dc217c-8f81-4b1c-b42e-071ba2b8b4f6)